### DISCUSSION TITLE
What is the proper location for created Ansible installation playbook

### DISCUSSION THREAD

**Discussion by kwoot** [minvws/nl-kat-coordination#1613] [Show and tell]
Hi,

I am currently working on  Ansible installation (and upgrade) playbooks (1.10.1 on Debian 12). Almost done.
What would be the proper location in the repo to store these?
Usually a repo has a subdir called 'contrib'  for contributions like these to the project.
Somehow 'scripts'  does not seem a good fit for playbooks.
Maybe /contrib/playbooks?

Please advice.

**Comment by underdarknl** [comment]
I agree that contrib seems to be the defacto location for contributed scripts like these, and while the idea is that code in the contrib folder is of lesser concern to the repository owner, as it was contributed, and thus might not work / needs maintenance, I feel like we'll end up with the support questions about anything in a contrib folder anyway.
Since we would need to  support these scripts like any other piece of code as we are distributed them (or at least we will be expected to), I'd suggest to just put these in the scripts folder.

We should however make sure that we only accept code that we expect in maintainable, or will be maintained by the parties who contribute the code to us.

**Comment by kwoot** [comment]
You know a simple README in contrib could fix your worries, right? 

In the mean time, I will make a MR for scripts, so at least people can start using it if they want.

**Comment by underdarknl** [comment]
It should, but it never does, as requests for help will still end up primarily in this issue tracker, and reflect on the product that the contributed code is shipped with.
I'd also only want to ship stuff that works, which means we'd need to keep track of changes anyway, hopefully with help from the original authors where possible.

Looking forward to the PR, The contribution that triggered this question is very much wanted and very helpful. Thanks in advance.

**Reply by kwoot** [reply]
You're very welcome. I will try to make a MR this weekend.
